articles/getting-started-with-inheritance-using-c/index.md

```diff
@@ -0,0 +1,228 @@
+
+```c#
+    public class Point
+    {
+      private int x, y;
+
+      public Point( int xcoordinate, int ycoordinate )
+      {
+         X = xcoordinate;
+         Y = ycoordinate;
+         Console.WriteLine("point constructor: {0}", this);}
+  ```
+
+Here we define the set and get Accessors to access the private members of the class, then we'll override the ToString method.
+
+**Note** that the ToString method will return a string representation of the class Point.
+
+  ```c#
+      public int X
+      {
+         get
+         {return x;}
+         set
+         {x = value;}
+      }
+
+      public int Y
+      {
+         get
+         {return y;}
+         set
+         {y = value;}
+      }
+
+      public override string ToString()
+      {return "[" + X + ", " + Y + "]";}
+    } // end class Point
+   ```
+
+The following class is the Derived class of the class point.
+
+In the class Circle, we don't need to define the point coordinates because they've been inherited.
+
+`: base(xcoordinate,ycoordinate)`.
+
+Now we'll define the radius of the circle, the constructor, and the Accessors.
+
+   ```c#
+       public class Circle: Point
+    {
+       private double radius;
+
+      public Circle( int xcoordinate, int ycoordinate, double radiuscircle )
+         : base( xcoordinate, ycoordinate )
+      {Radius = radiuscircle;
+      Console.WriteLine("Circle constructor: {0}", this);}
+
+      public double Radius
+      {
+         get
+         {return radius;}
+         set
+         {radius = value;}
+      }
+
+   ```
+
+Now we need to define the functions that will help us calculate the area and perimeter of the circle, then we'll override the ToString method, just like we did in the class Point.
+
+   ```c#
+      public double Diameter()
+      {return Radius * 2;}
+
+      public double Perimeter()
+      {return Math.PI * Diameter();}
+
+
+      public virtual double Area()
+      {return Math.PI * Math.Pow( Radius, 2 );}
+
+      // Return string representation of Circle
+      public override string ToString()
+      {
+      // Using the ToString method with extra additions
+         return "Center= " + base.ToString() +"; Radius = " + Radius;   
+      }
+   } // end class Circle
+   ```
+
+Let's take a look at the following code:
+
+   ```c#
+   static void Main(string[] args)
+        {
+            Circle c = new Circle(6,8,4);
+
+            Console.WriteLine("Area : " + c.Area());
+            Console.WriteLine("Perimeter : " + c.Perimeter());
+        }
+   ```
+
+The output after execution should look like:
+
+   ```c#
+   point constructor: Center= [6, 8]; Radius = 0
+   Circle constructor: Center= [6, 8]; Radius = 4
+   Area : 50.2654
+   Perimeter : 25.1327
+  ```
+
+### Constructors and Destructors in derived classes
+The constructor of a base class will be called implicitly or explicitly when Instantiating a derived class, this will cause a chain reaction when a base class is a derived class as well.
```

Comment:
Instantiating -> instantiating. Use italics to draw the attention of the reader instead of capitalization midway in the sentence.